### PR TITLE
Customizable center point of overlay component

### PR DESCRIPTION
**Pull request by krris**
`HubViewControllerDelegate` can manage the center point of overlay components.

### CODE REVIEW

**Comment by krris** [comment]
Ping 🎾  @JohnSundell @nataliq

File: sources/HUBViewController.m [hunk 1]
```diff
@@ -1051,7 +1051,10 @@ - (CGPoint)overlayComponentCenterPoint
     CGRect frame = self.view.bounds;
     frame.origin.y = self.collectionView.contentInset.top;
     frame.size.height -= self.visibleKeyboardHeight + CGRectGetMinY(frame);
-    return CGPointMake(CGRectGetMidX(frame), CGRectGetMidY(frame));
+
+    CGPoint proposedCenterPoint = CGPointMake(CGRectGetMidX(frame), CGRectGetMidY(frame));
+    return [self.delegate centerPointForOverlayComponentInViewController:self
```
Comment (JohnSundell) [inline sources/HUBViewController.m]: If the delegate is `nil` this will return `CGPointZero`, which is not what we want :)

Comment (krris) [inline sources/HUBViewController.m]: Right. I'll fix that.

**Comment by krris** [comment]
@JohnSundell 🎾

**Pull request merged**